components/Calendar/ReactCalendar.tsx

```diff
@@ -0,0 +1,186 @@
+                            <svg
+                                xmlns="http://www.w3.org/2000/svg"
+                                fill="none"
+                                viewBox="0 0 24 24"
+                                strokeWidth={2}
+                                stroke="currentColor"
+                                className="w-6 h-6 mr-1"
+                            >
+                                <path strokeLinecap="round" strokeLinejoin="round" d="M15 19l-7-7 7-7" />
+                            </svg>
+                            Назад кон време
+                        </button>
+                        <form onSubmit={handleFormSubmit} className="flex flex-col items-center gap-2 mt-4 w-10/12">
+                            <label htmlFor="race" className="w-full justify-start flex text-black">Раса на куче</label>
+                            <input
+                                id="race"
+                                type="text"
+                                placeholder="Доберман"
+                                className="text-base py-1 pl-4 border border-gray rounded-4xl text-black w-full mt-1 mb-3"
+                                required
+                            />
+
+                            <label htmlFor="select1" className="w-full justify-start flex text-black">Тип на прошетка</label>
+                            <select
+                                id="select1"
+                                className="py-1 pl-4 border border-gray rounded-4xl text-black w-full mt-1 mb-3"
+                                required
+                            >
+                                <option value="" disabled selected hidden>Социјализација</option>
+                                <option value="option1">Option 1</option>
+                                <option value="option2">Option 2</option>
+                                <option value="option3">Option 3</option>
+                            </select>
+
+                            <label htmlFor="select2" className="w-full justify-start flex text-black">Времетраење на прошетка</label>
+                            <select
+                                id="select2"
+                                className="py-1 pl-4 border border-gray rounded-4xl text-black w-full mt-1 mb-3"
+                                required
+                            >
+                                <option value="" disabled selected hidden>1:30 минути</option>
+                                <option value="option1">Option 1</option>
+                                <option value="option2">Option 2</option>
+                                <option value="option3">Option 3</option>
+                            </select>
+
+                            <button
+                                type="submit"
+                                className="bg-orange text-white p-2 rounded-xl mt-9 w-full"
+                            >
```

Comment:
Button color should change if the form is filled or not.
![image](https://github.com/user-attachments/assets/4108ddba-5be7-4093-ab2f-bce08d4fed89)
![image](https://github.com/user-attachments/assets/bbf78730-0e53-42f3-b890-3b70d4a32de1)